src/renderer/components/editorWithTabs/editor.vue

```diff
@@ -590,6 +590,12 @@ export default {
           animatedScrollTo(container, container.scrollTop + y - STANDAR_Y, 100)
         }
 
+        // Used to fix #628: auto scroll cursor to visible if the cursor is too low.
+        if (container.clientHeight - y < 100) {
+          // 740 is the lowest cursor position(till to top) that editor allowed.
+          animatedScrollTo(container, container.scrollTop + (y - 740), 0)
```

Comment:
That magic seems to work but scrolling to top doesn't work and there was an exception during testing but I don't know why and how to reproduce it:

```
TypeError: Cannot read property 'nodeType' of null
    at getNodeAndOffset (webpack-internal:///./src/muya/lib/selection/index.js:438:16)
    at Selection.setCursorRange (webpack-internal:///./src/muya/lib/selection/index.js:512:9)
    at ContentState.setCursor (webpack-internal:///./src/muya/lib/contentState/index.js:228:56)
    at ContentState.partialRender (webpack-internal:///./src/muya/lib/contentState/index.js:297:12)
    at ContentState.arrowHandler (webpack-internal:///./src/muya/lib/contentState/arrowCtrl.js:260:19)
    at HTMLDivElement.handler (webpack-internal:///./src/muya/lib/eventHandler/keyboard.js:218:26)
```

Reply:
The magic number is according to window/editor height, I'll change it...